### PR TITLE
Update to Yocto 2.1

### PR DESCRIPTION
**Pull request by williamspatrick**
This change is [Reviewable](https://reviewable.io/reviews/openbmc/openbmc/396)

### CODE REVIEW

**Comment by williamspatrick** [comment]
Do not merge yet.

**Comment by williamspatrick** [comment]
Implements #383

**Comment by williamspatrick** [comment]
Ready for review / merge.

File: meta-phosphor/classes/obmc-phosphor-image.bbclass [hunk 1]
```diff
@@ -48,9 +48,12 @@ CORE_IMAGE_EXTRA_INSTALL_append = " bash \
...
-
-inherit ${@build_overlay(d)}
+def image_overlay_enabled(d, fstype):
+        if d.getVar('OBMC_PHOSPHOR_IMAGE_OVERLAY', True) != "1":
+            return ""
+        if fstype:
+            return "overlay"
+        return "image-overlay"
+
+IMAGE_FSTYPES += "${@image_overlay_enabled(d, True)}"
```
Comment (kc8apf) [inline meta-phosphor/classes/obmc-phosphor-image.bbclass]: Using the 2nd arg to toggle between fixed use cases (IMAGE_FSTYPES and class to inherit from) is ugly.  Maybe take a string that is used as the value to return on match similar to bb.utils.contains()?

**Comment by shenki** [comment]
Woot! Nice work Patrick. Hopefully keeping in line with Yocto releases will be easier from here on in.

**Pull request merged**